api/views.py

```diff
@@ -416,3 +418,20 @@ class Accounts(generics.RetrieveAPIView):
     def get(self, request):
         serializer = AccountSerializer(request.user)
         return response.Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ContactUs(generics.CreateAPIView):
+    serializer_class = ContactUsSerializer
+    permission_clsses = permissions.AllowAny
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+        return response.Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+class Feedback(generics.CreateAPIView):
+    serializer_class = FeedbackSerializer
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+        return response.Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
```suggestion
```
This method definition is redundant. This basic implementation is provided by default in `generics.CreateAPIView`

Reply:
Since it is redundant, if I remove this what should I do then?

Reply:
You don't need to do anything except remove it. Removing the method should have the same effect as if it were there. Take a look at the `generics.CreateAPIView` source and you will find that the default implementation defines a post method that simply passes the data from the request to the serializer specified by the `serializer_class` class attribute and then saves the data.